Add icon for Computherm B series

zadoli opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Proposed change
<!-- 
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request.
-->


## Type of change
<!--
  What type of change does your PR introduce to the Home Assistant Brands?
  NOTE: Please, check only 1! box! 
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [x] I've added a link to my custom integration repository in the PR description
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Replace an existing icon or logo after a branding change
- [ ] Removing an icon or logo

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- Link to custom integration repository: https://github.com/zadoli/ha-computherm-b

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your contribution.
-->

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

<!--
  Thank you for contributing <3
-->

frenck reviewed (changes requested):
Oh hi there @zadoli 👋 

Thanks for the pull request. It seems that the provided branding is modified specifically to target a product? (The b series part).

The branding on their website seems different though:

![CleanShot 2025-02-10 at 14 38 46](https://github.com/user-attachments/assets/2263a084-5860-4132-8534-aa52b0042875)

Could you take a look?

../Frenck

frenck reviewed (changes requested):
I couldn't find this icon being used anywhere? 

Can you point me to where the manufacturer uses this as their branding for their icons?

Thanks! 👍 

../Frenck

zadoli commented:
Hi @frenck!

My integration focusing only on B series product of Computherm, so this is why I choed this icon, but now I modified the logo based on the original, which I got from [Wikimedia](https://commons.wikimedia.org/w/index.php?search=computherm&title=Special:MediaSearch&type=image) It is used on the [page](https://computherm.info/en) which you also mentioned before. I changed the color to match the icon for B Series. Is this problem?

 Cloud platform for B series: https://computhermbseries.com/

frenck commented:
So, the thing is, we collect manufacturer branding, not product branding. We also cannot accept modified versions of branding assets, as they often violate upstream branding guidelines.

Please use the original manufacturer branding instead.

../Frenck

frenck reviewed (approved):
Thanks, @zadoli 👍 

../Frenck
